### PR TITLE
Changelog 140

### PR DESCRIPTION
**Pull request by vlad20012**

### CODE REVIEW

File: _posts/2021-02-01-changelog-140.markdown [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6609 -->
+* Download stdlib dependencies into IDE system directory instead of `src`
+  directory in toolchain location to avoid unexpected behavior like read-only directory.
+  Note, stdlib dependencies are downloaded by the plugin only when
+  `org.rust.cargo.fetch.actual.stdlib.metadata` experimental feature is enabled,
+  otherwise the plugin uses hardcoded stdlib structure
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6580 -->
+* Fix `Change item visibility` intention so that it is not offered in trait impl blocks. (by [@Kobzol])
```
Comment (ortem) [inline _posts/2021-02-01-changelog-140.markdown:92]:
```suggestion
* Do not suggest `Change item visibility` intention in trait impl blocks (by [@Kobzol])
```

**Review by Undin — CHANGES_REQUESTED** [review]

**Review by Undin — COMMENTED** [review]
Oh, and let's add note about dropping 2020.2 support in next release
Something like "This is the last release that supports the 2020.2 platform. To receive plugin updates further, please upgrade your IDE to 2020.3."

File: _posts/2021-02-01-changelog-140.markdown [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+* Add `Change Signature` refactoring (`Refactor | Change Signature` or <kbd>Ctrl</kbd>+<kbd>F6</kbd>)
+  for functions and methods (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-140/5944" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6563 -->
+* Introduce `Share in Playground` action to share your code on https://play.rust-lang.org/.
+  You can invoke it via `Tools | Rust | Share in Playground` or via context menu
+
+  {% include gif-img.html path="/assets/posts/changelog-140/6563" w="700px" %}
```
Comment (Undin) [inline _posts/2021-02-01-changelog-140.markdown:21]: Let's create a new gif where confirmation dialog is shown as well

**Review by Undin — APPROVED** [review]
LGTM

**Pull request merged**